templates/election_vote_results/_form_layout.html.twig

```diff
@@ -0,0 +1,91 @@
+{% extends 'election_vote_results/_layout.html.twig' %}
+
+{% block vote_results_content %}
+    {{ form_start(form, { attr: { class: 'em-form' } }) }}
+
+    <div class="text--right b__nudge--bottom-small">
+        <button class="btn btn--blue--soft" type="submit">Sauvegarder</button>
+    </div>
+
+    {% if vote_result.updatedBy is not empty %}
+        <p class="text--small text--silver-gray text--right">
+            dernière modification effectuée par <strong>{{ app.user == vote_result.updatedBy ? 'vous' : vote_result.updatedBy.fullName }}</strong>
+
+            {% if vote_result.updatedAt is not empty %}
+                , le {{ vote_result.updatedAt|localizeddate('none', 'none', 'fr_FR', null, "d MMM yyyy HH:mm") }}
```

Comment:
should the pattern looks like `"d MMM \à yyyy HH:mm"` (adding `\à`)

so it would be `le 15 mars 2020 à 19h30`

Reply:
👍